### ISSUE TITLE
snowflake `copy grants` fails to parse

### ISSUE THREAD

**Issue by vedas2** [tobymao/sqlglot#1791]
I have a SQL query like 
```
alter session set ABORT_DETACHED_QUERY = TRUE;
create or replace view my_view
(
    "USER_ID", 
    "GUEST_IND",
    "FIRST_ORDER_DATE_TIME_UTC"  
) 
copy grants
AS (
SELECT
    USER_ID,
    GUEST_IND,
    FIRST_ORDER_DATE_TIME_UTC
FROM
    (select * from __dbt__cte__user_base LIMIT 0)  
  );
  ```
  This fails to parse with error:
  ```
  sqlglot.errors.ParseError: Invalid expression / Unexpected token. Line 9, Col: 1.
  ate or replace view my_view
(
    "USER_ID", 
    "GUEST_IND",
    "FIRST_ORDER_DATE_TIME_UTC"  
) 
copy grants
AS (
SELECT
    USER_ID,
    GUEST_IND,
    FIRST_ORDER_DATE_TIME_UTC
FROM
    (select * fro
    ```
    
If I remove `copy grants` from the query it works. 
Since this is a valid snowflake query, it should be supported.
The problem differs from the original issue reported, I can create a new issue.

_Originally posted by @vedas2 in https://github.com/tobymao/sqlglot/discussions/1787#discussioncomment-6198754_

**Comment by vedas2** [comment]
Which version should I use to get this fix?

**Comment by GeorgeSittas** [comment]
There isn't a version that contains these fixes yet. You can either pull the repo and do a manual installation or wait until the new release (we make releases frequently)

**Comment by GeorgeSittas** [comment]
@vedas2 just released v16.3.0 which contains the fix.

**Comment by vedas2** [comment]
👍 Thank you for resolving this so quick.
